### ISSUE TITLE
Redefining i18n t results in always displaying wordIsACommonPassword error message

### ISSUE THREAD

**Issue by vitanivanov** [ablanco/jquery.pwstrength.bootstrap#148]
The documentation states:

> If you want to manage translations yourself or you don't use i18next you can override the default translation function like this:
> ```javascript
> $(document).ready(function () {
>     var options = {};
>     options.i18n = {
>         t: function (key) {
>             var result = translateThisThing(key); // Do your magic here
> 
>             return result === key ? '' : result; // This assumes you return the
>             // key if no translation was found, adapt as necessary
>         }
>     };
>     $(':password').pwstrength(options);
> });
>````

Here's what I do:
(translation is in Bulgarian)
```javascript
var password_clone = $("#password_clone");

function translateThisThing(key) {
    var translation = {
        "wordNotEmail": "Паролата трябва да е различна от имейла.",
        "wordSimilarToUsername": "Паролата трябва да е различна от потребителското име.",
        "wordTwoCharacterClasses": "Паролата трябва да е разнообразна - бувки и символи/числа",
        "wordRepetitions": "Паролата съдържа прекалено много повторения",
        "wordSequences": "Паролата съдържа последователни символи",
        "wordIsACommonPassword": "Паролата се използва прекалено често",
        "errorList": "Грешки:",
        "veryWeak": "Много слаба",
        "weak": "Слаба",
        "normal": "Нормална",
        "medium": "Силна",
        "strong": "Много силна",
        "veryStrong": "Желязна",
        "wordMinLength": "Паролата е под 6 символа",
    };
    return translation[key] || key;
}
var options = {
    i18n: {
        t: function(key) {
		'use strict';
		var result = translateThisThing(key); // Do your magic here
		return result === key ? '' : result; // This assumes you return the
		// key if no translation was found, adapt as necessary
        }
    },
    ui: {
        raisePower: 1.1,
        scores: [5, 25, 45, 60, 80],
        showVerdictsInsideProgressBar: true,
        showErrors: true,
        colorClasses: ["danger", "danger", "warning", "warning", "success", "success"]
    }
};
password_clone.pwstrength(options);
```
If I use my translation, even though wordIsACommonPassword is logged as "false", I get the error message shown inside the errors list.
If I comment the options.i18n redefinition, it works as expected.
Any ideas why?

**Comment by ablanco** [comment]
I've recreated your example and it seems to work fine:

https://codepen.io/anon/pen/gvKWEr?editors=1010#0

**Comment by vitanivanov** [comment]
It "works" but not as expected. Check the screenshot - I made the input "text" for visibility
Just type anything in the password input box - you'll see the wordIsACommonPassword error listed even though the rule evaluates as "false"
![example](https://user-images.githubusercontent.com/3907237/36528682-9c5e200c-17be-11e8-9eb5-a3c65dda915b.png)

**Comment by ablanco** [comment]
I understand it now. It looks like a bug, I'll try to fix it this weekend.

**Comment by ablanco** [comment]
Fixed in ac78ea494f1249d192993ef3846e571e67556aa7
